admin/app/controllers/admin/InteractiveLibrarianController.scala

```diff
@@ -18,6 +18,28 @@ class InteractiveLibrarianController(
     with GuLogging
     with ImplicitControllerExecutionContext {
 
+  def pressForm(): Action[AnyContent] =
+    Action { implicit request =>
+      // This view is displayed on the admin tool
+      Ok(views.html.pressInteractive())
+    }
+
+  def press(path: String): Action[AnyContent] =
+    Action.async { implicit request =>
+      // This function combines both pressing and cleaning
```

Comment:
If the comment is to document the method itself, I'd recommend putting it above the `def press...` line as then it will come up when people hover over the function elsewhere. Note, you need to use this comment style for this:

    /**
      * This function...
      */